packages/app/src/app/graphql/client.js

```diff
@@ -24,14 +24,11 @@ const authLink = setContext((_, { headers }) => {
 });
 
 const absintheAfterware = new ApolloLink((operation, forward) =>
-  forward(operation).map(result => {
-    /* eslint-disable no-param-reassign */
-    result.errors = result.payload.errors;
-    result.data = result.payload.data;
-    /* eslint-enable */
-
-    return result;
-  })
+  forward(operation).map(({ payload, ...result }) => ({
```

Comment:
@Saeris Why didn't you further destructur `payload` here?

Reply:
I didn't run this locally, I made the edit quickly here on GitHub, so I don't know what else is included in the payload object from Absinthe, or if anything else from payload is actually getting used somehow (I don't believe Apollo Client uses it).

If there is a specific reason to forward it, I'd be interested in hearing it. Otherwise this should not be a breaking change.

Reply:
You only use `data` & `errors ` from `payload`, so you could just have done it like this

```js
.map(({ payload: { data, errors }, ...result }) => ({
  ...result,
  data,
  errors,
})
```